lib/rules/no-context.js

```diff
@@ -75,6 +87,15 @@ module.exports = {
 
     return {
 
+      ClassProperty: function(node) {
```

Comment:
Not entirely sure. Since that's not part of standard JS yet (still stage 2), you might have to jump through some extra hoops to make sure there is a parser that can understand that experimental syntax.

In astexplorer.net it looks like espree fails on this syntax, but babel-eslint does not. So you probably need to specify `parser: 'babel-eslint'` in your test for these.